### DISCUSSION TITLE
Unable to find the configure module in source

### DISCUSSION THREAD

**Discussion by vijiv6190** [OpenIDC/ngx_openidc_module#16] [Q&A]
Unable to execute below command when trying to configure the dynamic module. *.so modules are not available in the source code
sudo ./configure --add-dynamic-module=.

I am trying to integrate openidc with nginx. Please provide the configuration details to add openidc module to nginx.

**Comment by zandbelt** [comment]
see here https://github.com/OpenIDC/ngx_openidc_module/blob/master/.github/workflows/build.yml or better use a package from the Assets section of https://github.com/OpenIDC/ngx_openidc_module/releases

**Reply by vijiv6190** [reply]
I have followed the instructions from https://github.com/OpenIDC/ngx_openidc_module/blob/master/.github/workflows/build.yml and ran the configure command. Unable to restart nginx and getting below error.
I have lua package installed on the server as well.

 Error : nginx: [emerg] unknown directive "lua_package_path" in /etc/nginx

I am using below configuration in nginx.conf for redirecting to openIDC

access_by_lua_block {
      local opts = {
        redirect_uri = "<redirect_uri>",
        discovery = "<cloudsso_discovery_url>",
        client_id = "<oauth_client_id>",
        client_secret = "<oauth_client_secret>",
        ssl_verify = "no",
        accept_none_alg = false,
        accept_unsupported_alg = false,
        renew_access_token_on_expiry = true,
        access_token_expires_in = 3600,
        revoke_tokens_on_logout = true,
      }
       local res, err = require("resty.openidc").authenticate(opts)
      if err then
        ngx.status = 500
        ngx.say(err)
        ngx.exit(ngx.HTTP_INTERNAL_SERVER_ERROR)
      end
       ngx.req.set_header("sub", res.id_token.sub)
     }
.
}

**Comment by zandbelt** [comment]
seems you're confusing two different projects: ngx_openidc_module and lua-resty-openidc

**Reply by vijiv6190** [reply]
can you please suggest nginx.conf using openidc_module
